src/components/conducted-language/conducted-language.html

```diff
@@ -0,0 +1,9 @@
+<ion-row *ngIf="shouldRender">
+  <ion-col></ion-col>
```

Comment:
Do we need this first column?

Reply:
No, it's redundant I guess. It was to spare a huge amount of padding. I will remove. 👍